repl/src/main/scala/com/cloudera/livy/repl/PythonInterpreter.scala

```diff
@@ -37,8 +37,11 @@ import com.cloudera.livy.client.common.ClientConf
 
 // scalastyle:off println
 object PythonInterpreter extends Logging {
-  def apply(conf: SparkConf): Interpreter = {
-    val pythonExec = sys.env.getOrElse("PYSPARK_DRIVER_PYTHON", "python")
+  def apply(conf: SparkConf, kind: String = "pyspark"): Interpreter = {
+    val pythonExec = {
+        if (kind == "pyspark3") sys.env.getOrElse("PYSPARK3_DRIVER_PYTHON", "python3")
```

Comment:
So not only you're using a string but you're using it as a boolean... it'd be better to just use a boolean from the get go, or do proper matching (and throw an error if the wrong value is provided).